### ISSUE TITLE
this does not work on iOS Chrome/ Safari

### ISSUE THREAD

**Issue by wilfredonoyola** [CookPete/react-player#706]
#### Current Behavior

When I tried to test on mobile version for my website on iOS Device and I am using the navigator google chrome and Safari. Both does not display the player, this only shows a placeholder the player.

#### Expected Behavior

This will work correctly on iOS devices.

#### Steps to Reproduce

1. You need a iOS device
1.  Navigate to you player

#### Environment
- URL attempting to play:  Video
- Browser: chrome and Safari
- Operating system: ios

**Comment by killvung** [comment]
Have you clicked any of the testing video provide in the demo. For example, YouTube has the button `Test A`, `Test B` and `Playlist`

**Comment by CookPete** [comment]
I can't help you without more information. What URL are you trying to play? As @killvung has said, the demo page does not play a video automatically – you must select a type of video to play first.

**Comment by killvung** [comment]
Oh man, I love how people here can raise questions freely without sufficient information. If I do thing at my work I will get yell at instantly...

**Comment by killvung** [comment]
Oh man, I love how people here can raise questions freely without sufficient information. If I do this at my work I will get yell at instantly...

**Comment by memorx** [comment]
you should change jobs

**Comment by joepagan** [comment]
I believe this is still an issue. I know it is somewhat referenced here: https://github.com/CookPete/react-player#mobile-considerations

But, I believe that the behaviour described is acurate, though I think OP should have made it more obvious they were attempting to play a file.

Basically, in firefox, chrome & safari on ios the video does not start at all, even when you double tap it (as suggested as the solution on various issues I have seen).

Anyway, the [react-player demo](https://cookpete.com/react-player/) does behave this way too. Double tapping the video element does not start a video played from a file, but, the play button does load the file & start playing it.

I believe I have come up with a part-way solution:
https://codesandbox.io/s/modern-frog-nxz4k

Basically, for mobile, I think the solution is to use the `light` param so that it's obvious the element is a video & can be played. This seems to work.
You could use https://github.com/duskload/react-device-detect to set the `light` param with a conditional too just for mobile users.

**Comment by CookPete** [comment]
> Seems like a good idea to set a poster & a light param.

Thanks for this. I'll add it to the readme section.

**Comment by joepagan** [comment]
No probs, thanks for your work 👍
